net-analyzer/netdata/netdata-2.5.3.ebuild

```diff
@@ -92,26 +100,46 @@ pkg_setup() {
 	linux-info_pkg_setup
 }
 
-PATCHES=(
-	"${FILESDIR}"/${PN}-dlib-global_optimization-add-template-argument-list.patch
-)
+#PATCHES=(
+#	"${FILESDIR}"/${PN}-dlib-global_optimization-add-template-argument-list.patch
+#)
 
```

Comment:
```suggestion
```

You can just remove it entirely, this change is in 19.24.8.
https://github.com/davisking/dlib/commit/0947215ddb268fef192e45c140c9430f77648166